README.md

```diff
@@ -724,885 +711,586 @@ signIn.after('identify', async () => {
 
 ## Configuration
 
-For non-OIDC applications, the only required configuration option is `baseUrl`. All others are optional.
+If you are using the [default Okta-hosted signin page](#okta-hosted-sign-in-page-default), all configuration is handled via the `Customization` section of the Admin UI.
 
-```javascript
-// Basic example
-var config = {
-  baseUrl: 'https://{yourOktaDomain}',
-  logo: '/path/to/logo.png',
-  logoText: 'logo text',
-  helpSupportNumber: '(123) 456-7890',
-  language: 'en',
-  i18n: {
-    en: {
-      'primaryauth.title': 'Sign in to Acme'
-    }
-  },
-  helpLinks: {
-    help: 'https://acme.com/help'
-  },
-  authParams: {
-    // Configuration for the internal authClient. See https://github.com/okta/okta-auth-js#configuration-options
-  }
-};
+If you are using the [custom Okta-hosted signin page](#okta-hosted-sign-in-page-customizable), a configuration object is included on the page which contains all necessary values. You will probably not need to modify this object, but you may use this object as a starting point and add additional customizations.
 
-var signIn = new OktaSignIn(config);
-```
+For embedded widgets, you should set the `issuer`, `clientId`, and `redirectUri`. Additionally you should set `useInteractionCodeFlow` to `true` to enable the OIE pipeline. (See [this document](https://github.com/okta/okta-signin-widget/blob/master/docs/classic.md) for details on running in "classic" pipeline).
```

Comment:
we should probably change in this file "classic pipeline" to "classic engine", right? There are a few instances

Reply:
and maybe "OIE pipeline" too? 🤔